api/tests/integration/infrastructure/utils/csv/certification-results_test.js

```diff
@@ -1,59 +1,119 @@
+          `"${certifResult1.id}";"Lili";"${certifResult1.lastName}";"${expectedBirthDate}";"${certifResult1.birthplace}";${certifResult1.externalId};${certifResult1.pixScore};0;1;5;"-";"-";"-";"-";"-";"-";"-";"-";"-";"-";"-";0;0;${session.id};"${session.certificationCenter}";"${expectedCreatedAt}"\n` +
+          `"${certifResult2.id}";"Tom";"${certifResult2.lastName}";"${expectedBirthDate}";"${certifResult2.birthplace}";${certifResult2.externalId};"0";"-";"-";"-";"-";"-";"-";"-";"-";"-";"-";"-";"-";"-";"-";0;0;${session.id};"${session.certificationCenter}";"${expectedCreatedAt}"`;
+        expect(result).to.deep.equal(expectedResult);
       });
+    });
 
-      const certifResult1 = domainBuilder.buildCertificationResult({
-        lastAssessmentResult: lastAssessmentResult1,
-        firstName: 'Lili',
-        birthdate,
-        createdAt,
-      });
-      const certifResult2 = domainBuilder.buildCertificationResult({
-        lastAssessmentResult: lastAssessmentResult2,
-        firstName: 'Tom',
-        birthdate,
-        createdAt,
+    describe('when at least one certification has passed CleA certification', () => {
+
+      it('should return correct csvContent with the CleA information', async () => {
+        // given
+        const certificationCenterId = databaseBuilder.factory.buildCertificationCenter({}).id;
+        const session = databaseBuilder.factory.buildSession({ certificationCenterId });
+
+        const birthdate = new Date('1990-01-01');
+        const createdAt = new Date('2020-01-01');
+
+        const competencesWithMark1 = [
+          { competence_code: '1.1', level: 0 }, { competence_code: '1.2', level: 1 }, { competence_code: '1.3', level: 5 },
+          { competence_code: '5.1', level: 0 }, { competence_code: '5.2', level: -1 },
+        ];
+        const competencesWithMark2 = [ { competence_code: '5.1', level: 3 }, { competence_code: '5.2', level: -1 } ];
+
+        const lastAssessmentResult1 = domainBuilder.buildAssessmentResult({
+          status: 'validated',
+          competenceMarks: competencesWithMark1,
+        });
+        const lastAssessmentResult2 = domainBuilder.buildAssessmentResult({
+          status: 'rejected',
+          competenceMarks: competencesWithMark2,
+        });
+
+        const certifResult1 = domainBuilder.buildCertificationResult({
+          lastAssessmentResult: lastAssessmentResult1,
+          firstName: 'Lili',
+          birthdate,
+          createdAt,
+          cleaCertificationStatus: cleaStatuses.NOT_PASSED,
+        });
+        const certifResult2 = domainBuilder.buildCertificationResult({
+          lastAssessmentResult: lastAssessmentResult2,
+          firstName: 'Tom',
+          birthdate,
+          createdAt,
+          cleaCertificationStatus: cleaStatuses.REJECTED,
```

Comment:
😈 Moi je parie que si on remplace par :
```suggestion
          cleaCertificationStatus: cleaStatuses.ACQUIRED,
```
en remplacant par ci par là dans le test les impacts, le test ne marche plus 😈 😈 😈